docs/API.md

```diff
@@ -81,151 +119,394 @@ Jest uses Jasmine 2 by default. An introduction to Jasmine 2 can be found
+```
+{
+  times: number;   // The number of test cases to run. Default: 100.
+  maxSize: number; // The maximum size of sized data such as numbers
+                   // (their magnitude) or arrays (their length). This can be
+                   // overridden with `gen.resize`. Default: 200.
+  seed: number;    // The random number seed. Defaults to a random value.
+}
+```
 
-Examples of dependencies that might be considered "implementation details" are things ranging from language built-ins (e.g. Array.prototype methods) to highly common utility methods (e.g. underscore/lo-dash, array utilities etc) and entire libraries like React.js.
+### `check.fit(name, [options], generators, fn)`
 
-*Note: this method was previously called `autoMockOff`. When using `babel-jest`, calls to `disableAutomock` will automatically be hoisted to the top of the code block. Use `autoMockOff` if you want to explicitly avoid this behavior.*
+Executes this test and skips all others. Like `fit`, but for property tests.
 
-### `jest.enableAutomock()`
-Re-enables automatic mocking in the module loader.
+### `check.xit(name, [options], generators, fn)`
 
-*Note: this method was previously called `autoMockOn`. When using `babel-jest`, calls to `enableAutomock` will automatically be hoisted to the top of the code block. Use `autoMockOn` if you want to explicitly avoid this behavior.*
+Skips this test. Like `xit`, but for property tests.
 
-### `jest.fn(?implementation)`
-Returns a new, unused [mock function](#mock-functions). Optionally takes a mock
-implementation.
+### `gen`
+
+A library of generators for property tests. See the
+[`testcheck` documentation](https://github.com/leebyron/testcheck-js).
+
+### `require.requireActual(moduleName)`
+
+Returns the actual module instead of a mock, bypassing all checks on whether the
+module should receive a mock implementation or not.
+
+### `require.requireMock(moduleName)`
+
+Returns a mock module instead of the actual module, bypassing all checks on
+whether the module should be required normally or not.
+
+## Writing assertions with `expect`
+
+### `expect(value)`
+
+If you haven't used `expect` before, you might find its behavior a little unintuitive. You will rarely call `expect` by itself. Instead, you will use `expect` along with a "matcher" function to assert something about a value.
+
+It's easier to understand this with an example. Let's say you have a method `bestLaCroixFlavor()` which is supposed to return the string `'grapefruit'`.
+Here's how you would test that:
 
 ```js
-  const mockFn = jest.fn();
-  mockFn();
-  expect(mockFn).toBeCalled();
+describe('the best La Croix flavor', () => {
+  it('should be grapefruit', () => {
```

Comment:
the conventions that most of the people seem to follow is to not use `should` in test descriptions.
so that line would look like `it('is grapefruit', () => {})`